alias/gui.c

```diff
@@ -85,20 +85,30 @@ int alias_sort_address(const void *a, const void *b)
     if (addr_a && addr_a->personal)
     {
       if (addr_b && addr_b->personal)
-        r = mutt_istr_cmp(addr_a->personal, addr_b->personal);
+        r = mutt_str_coll(addr_a->personal, addr_b->personal);
       else
         r = 1;
     }
     else if (addr_b && addr_b->personal)
       r = -1;
     else if (addr_a && addr_b)
-      r = mutt_istr_cmp(addr_a->mailbox, addr_b->mailbox);
+      r = mutt_str_coll(addr_a->mailbox, addr_b->mailbox);
     else
       r = 0;
   }
   return RSORT(r);
 }
 
+int alias_sort_unsort(const void *a, const void *b)
```

Comment:
Please can you add a Doxygen comment block for this function.

Reply:
Added.